docs/UserGuide.md

```diff
@@ -638,8 +648,15 @@ you can search for the customer using keywords by calling this command._
 
 **Note:**
 
-* You must provide at least one keyword to search for the customer.
-* You can choose to provide multiple keywords.
+* You must provide at least one non-empty alphanumeric `KEYWORD`{.swift} to search for the Customer.
+* You can optionally provide additional keywords, a Customer name that matches any of the given keywords will be 
+  displayed.
+  For example, the keywords `Alex Tan`{.swift} will display `Alex Wong`{.swift}, `Alex Tan`{.swift},
+  and `Tan Ah Meng`{.swift}.
+* The keyword must exactly match any word in the Customer name. For example the keyword `Alex`{.swift} will match
+  `Alex`{.swift} but not `Alexander`{.swift}.
+* The search for keyword(s) given are case-insensitive
```

Comment:
Should this be a statement with a full-stop? Also, could we say "The search is not case sensitive."